### PR TITLE
Business card update

### PR DESCRIPTION
**Pull request by kattni**
Added email display functionality to `show_business_card`, and changed `data` in `show_qr_code` to a string from a bytearray. Updated examples to be accurate and more thorough.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,17 +61,17 @@ from adafruit_pybadger import PyBadger
 pybadger = PyBadger()
 
 while True:
-    pybadger.show_badge(hello_scale=2, my_name_is_scale=2, name_scale=3)
+    pybadger.show_badge(name_string="Blinka", hello_scale=2, my_name_is_scale=2, name_scale=3)
```
Comment (makermelissa) [inline README.rst:65]: I would recommend moving show_badge and auto_dim out of the loop and then allowing something like pressing the start button to switch back to badge mode.

File: adafruit_pybadger.py [hunk 2]
```diff
@@ -227,9 +232,16 @@ def show_business_card(self, image_name=None, dwell=20):
...
-            time.sleep(dwell)
+        email_group = displayio.Group(scale=email_scale)
+        email_label = Label(email_font, text=email_string)
+        (_, _, width, height) = email_label.bounding_box
+        email_label.x = ((self.display.width // (2 * email_scale)) - width // 2)
+        email_label.y = int(height // (0.13 * email_scale))
+        email_label.color = 0xFFFFFF
+        email_group.append(email_label)
+        business_card_splash.append(email_group)
+        time.sleep(dwell)
```
Comment (makermelissa) [inline adafruit_pybadger.py]: I'm not entirely sure of the point of the automatically sleep for 20 seconds is. If this is for the purposes of the simepletest, would it be more useful to put it in the simpletest itself?

**Review by makermelissa — CHANGES_REQUESTED** [review]
The only other thing is that the business_card function only works in 4.1 Beta.

File: adafruit_pybadger.py (hunk 2 above)
Comment (kattni) [reply adafruit_pybadger.py]: I realised earlier that I need to make the code non-blocking, but didn't have the time to do so yet. Please do not merge until I have done that - I'll get to it this evening.

**Pull request merged**